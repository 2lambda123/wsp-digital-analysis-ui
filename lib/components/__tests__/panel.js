/* global describe, it, expect */

describe('Component > Panel', () => {
  const renderer = require('react-test-renderer')
  const React = require('react')
  const {Body, Heading, Panel} = require('../panel')

  describe('Panel', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Panel>Panel text</Panel>).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })

  describe('Heading', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Heading>Heading text</Heading>).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })

  describe('Body', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Body>Body text</Body>).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
