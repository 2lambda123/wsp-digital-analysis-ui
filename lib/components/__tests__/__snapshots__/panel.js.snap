// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > Panel Body renders correctly 1`] = `
<Body>
  <div
    className="panel-body"
  >
    Body text
  </div>
</Body>
`;

exports[`Component > Panel Heading renders correctly 1`] = `
<Heading>
  <div
    className="panel-heading clearfix "
    role="button"
  >
    Heading text
  </div>
</Heading>
`;

exports[`Component > Panel Panel renders correctly 1`] = `
<Panel>
  <div
    className="panel panel-default"
  >
    Panel text
  </div>
</Panel>
`;
